docs/codebase-best-practices.md

```diff
@@ -12,12 +12,12 @@ The `!important` keyword may be used to override values in some cases (for ex.:
 
 ### RTL support
 
-We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are a quick role of thumbs to follow:
+We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are some quick rules of thumb to follow:
```

Comment:
This seems to be change unrelated to the linked issue, please either remove commit making the change from PR, or revert the text changes.

```suggestion
We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are a quick role of thumbs to follow:
```

Reply:
Reverted the text changes and renamed the PR, please check it out!